dask/array/gufunc.py

```diff
@@ -625,19 +625,29 @@ class gufunc:
     .. [2] https://docs.scipy.org/doc/numpy/reference/c-api/generalized-ufuncs.html
     """
 
-    def __init__(self, pyfunc, **kwargs):
+    def __init__(
+        self,
+        pyfunc,
+        signature=None,
```

Comment:
Yeah that's a good point. I think it is probably good to be strict since it's easy to do. It's just like this right:

```suggestion
        *,
        signature=None,
```